extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -117,7 +117,9 @@ export class AmpConsent extends AMP.BaseElement {
 
     const config = new ConsentConfig(this.element);
 
-    this.consentConfig_ = config.getConsentConfig();
+    config.getConsentConfig().then(validatedConfig => {
```

Comment:
this can be problematic. we should change this.consentConfig_ to this.consentConfigPromise_. all the read of this.consentConfig_ shall wait for the promise.

Reply:
discussed offline